### PR TITLE
Session15 gaurav

### PR DESCRIPTION
**Pull request by gaur673**
Added one file related to the switch case statement

### CODE REVIEW

**Comment by Manisha5077** [comment]
Very Good Gaurav. Keep it up.

**Comment by gaur673** [comment]
Thanks .please share new T-shirt

**Review by Manisha5077 — APPROVED** [review]

File: src/Sesion15.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class Sesion15 {
+
+    public static void main(String[] args){
+        Sesion15 sesion15=new Sesion15();
+        sesion15.switchCase();
+
+    }
+
+    public void switchCase(){
+
```
Comment (Bharat-automation-test) [inline src/Sesion15.java:12]: @gaur673 please remove extra space from the file.

**Review by Bharat-automation-test — APPROVED** [review]

File: src/Sesion15.java (hunk 1 above)
Comment (gaur673) [reply src/Sesion15.java:12]: removed

**Review by Manisha5077 — APPROVED** [review]

**Review by Manisha5077 — APPROVED** [review]

**Review by Bharat-automation-test — APPROVED** [review]

**Comment by Jass0152** [comment]
HELLO

**Review by Manisha5077 — APPROVED** [review]

**Review by Bharat-automation-test — APPROVED** [review]

**Review by Manisha5077 — APPROVED** [review]

**Review by Manisha5077 — APPROVED** [review]

**Review by Bharat-automation-test — APPROVED** [review]

**Review by Bharat-automation-test — APPROVED** [review]

**Review by Bharat-automation-test — APPROVED** [review]

**Pull request merged**